components/menu/metadata/menu.yml

```diff
@@ -1135,38 +1135,38 @@ examples:
         <div class="spectrum-Examples-item">
           <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">Without icons</h4>
           <div class="spectrum-Examples-itemGroup">
-            <ul class="spectrum-Menu spectrum-Menu--sizeM is-selectableMultiple" role="menu">
+            <ul class="spectrum-Menu is-selectableMultiple" role="menu">
               <li class="spectrum-Menu-item is-selected" role="menuitem" tabindex="0">
                 <label class="spectrum-Checkbox spectrum-Checkbox--sizeM spectrum-Checkbox--emphasized spectrum-Menu-itemCheckbox">
-                  <input type="checkbox" class="spectrum-Checkbox-input" aria-labelledby="ms-label1" checked>
```

Comment:
Curious about the removal of `aria-labelledby` and their associated `id`s.

Reply:
The input and its label text are already wrapped inside the `<label>`, so they are not needed.

> _Alternatively, you can nest the `<input>` directly inside the `<label>`, in which case the for and id attributes are not needed because the association is implicit [[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)]_